tests/repros/repro28-pinheader-connections.test.tsx

```diff
@@ -0,0 +1,226 @@
+import { test, expect } from "bun:test"
+import { getTestFixture } from "../fixtures/get-test-fixture"
+import { sel } from "lib/sel"
+
+test(
+  "pinheader connections with netlabels",
```

Comment:
```suggestion
  "pinheader connections with netlabels",
```

describe what you're reproducing here

Reply:
Ok. The pin1 and pin4 are not getting connected of the JP1